packages/teleport/src/lib/tdp/codec.ts

```diff
@@ -609,6 +617,23 @@ export default class Codec {
     return buffer;
   }
 
+  // | message type (24) | completion_id uint32 | err_code uint32 |
+  encodeSharedDirectoryMoveResponse(res: SharedDirectoryMoveResponse): Message {
```

Comment:
What do you think of an alternative approach for encoding similar to https://github.com/gravitational/webapps/pull/1003#discussion_r931866959

Reply:
Great idea. I've added it as an issue: https://github.com/gravitational/webapps/issues/1087